core/src/main/scala/org/locationtech/rasterframes/expressions/generators/RasterSourceToTiles.scala

```diff
@@ -84,6 +84,8 @@ case class RasterSourceToTiles(children: Seq[Expression], bandIndexes: Seq[Int],
         Traversable.empty
     }
   }
+
+  override protected def withNewChildrenInternal(newChildren: IndexedSeq[Expression]): Expression = copy(children=newChildren)
```

Comment:
minor: seems the formatting is a bit off there `children=newChildren` => `children = newChildren`

Reply:
Heh, python formatting bleeding through.